recipe/meta.yaml

```diff
@@ -1,44 +1,50 @@
 {% set name = "mashumaro" %}
-{% set version = "3.8.1" %}
-
+{% set version = "3.6" %}
 
 package:
   name: {{ name|lower }}
   version: {{ version }}
 
 source:
-  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/mashumaro-{{ version }}.tar.gz
-  sha256: 8bae8b25e2287b75094655b8ba8635f34016c09ca16498188f2f3b198d88b7ef
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: ceb3de53029219bbbb0385ca600b59348dcd14e0c68523986c6d51889ad338f5
 
 build:
-  noarch: python
   number: 0
-  script: {{ PYTHON }} -m pip install . -vv
+  skip: True  # [py<37]
+  script: {{ PYTHON }} -m pip install . --no-deps --no-build-isolation -vv
 
 requirements:
   host:
     - pip
-    - python >=3.7
+    - python
+    - setuptools
+    - wheel
   run:
-    # Check for dependency changes at:
-    # <https://github.com/Fatal1ty/mashumaro/blob/v{{ version }}/setup.py>
-    - python >=3.7
+    - python
     - typing-extensions >=4.1.0
+    # Extra dependency is required by the downstream package dbt by default https://github.com/dbt-labs/dbt-core/blob/v1.5.3/core/setup.py#L57
```

Comment:
it is probably better to make dbt-core depend on both mashumaro and msgpack-python instead of forcing the optional dependency here

Reply:
Done!